testcases/bestest_air/models/find_days.py

```diff
@@ -6,28 +6,10 @@
 @author: dhbubu18
 """
 
-import pandas as pd
+from data.find_days import find_days
+import json
 
-csv_filepath = 'TestCase_Ideal.csv' # csv file containing model results (with Time as index column in seconds)
-heat = 'fcu.powHeaThe.y' # Column for heating load [W]
-cool = 'fcu.powCooThe.y' # Column for cooling load [W]
+days = find_days(heat='fcu_reaPCoo_y', cool='fcu_reaPHea_y', data='simulate')
```

Comment:
I think we would rather want the thermal load to be used instead of electrical or gas power, so as not to take COP changes (if they exist) into account.  The variables are as before:

heat = 'fcu.powHeaThe.y' # Column for heating load [W]	
cool = 'fcu.powCooThe.y' # Column for cooling load [W]

Reply:
Addressed in https://github.com/ibpsa/project1-boptest/pull/285/commits/d586aa78278bef0757a972ad538fbfa5ccb3f979.